tests/zfs-tests/include/tunables.cfg

```diff
@@ -35,6 +35,9 @@ DISABLE_IVSET_GUID_CHECK	disable_ivset_guid_check	zfs_disable_ivset_guid_check
 INITIALIZE_CHUNK_SIZE		initialize_chunk_size		zfs_initialize_chunk_size
 INITIALIZE_VALUE		initialize_value		zfs_initialize_value
 KEEP_LOG_SPACEMAPS_AT_EXPORT	keep_log_spacemaps_at_export	zfs_keep_log_spacemaps_at_export
+L2ARC_NOPREFETCH		UNSUPPORTED			l2arc_noprefetch
+L2ARC_WRITE_BOOST		UNSUPPORTED			l2arc_write_boost
+L2ARC_WRITE_MAX			UNSUPPORTED			l2arc_write_max
```

Comment:
I believe these options available on FreeBSD though I'm not positive what the exact names would be offhand.  Perhaps @freqlabs could comment on this.  That should allow your to make this test case common.

Reply:
```
l2arc.noprefetch
l2arc.write_boost
l2arc.write_max
```

Reply:
Done!